src/java.desktop/macosx/native/libawt_lwawt/awt/a11y/CommonComponentAccessibility.m

```diff
@@ -1253,3 +1257,50 @@ - (BOOL)accessibilityPerformIncrement {
                          waitUntilDone:NO];
     JNI_COCOA_EXIT(env);
 }
+
+/*
+ * Class:     sun_lwawt_macosx_CAccessible
+ * Method:    menuOpened
+ * Signature: (I)V
+ */
+JNIEXPORT void JNICALL Java_sun_lwawt_macosx_CAccessible_menuOpened
+    (JNIEnv *env, jclass jklass, jlong element)
+{
+    JNI_COCOA_ENTER(env);
+    [ThreadUtilities performOnMainThread:@selector(postMenuOpened)
+        on:(CommonComponentAccessibility *)jlong_to_ptr(element)
+        withObject:nil waitUntilDone:NO];
```

Comment:
Please move `waitUntilDone` to the new line for consistency.

Reply:
Fixed.